### DISCUSSION TITLE
[Bug?] Inconsistent form data with controlled field input

### DISCUSSION THREAD

**Discussion by gamino97** [react-hook-form/react-hook-form#6629] [Q&A] [answered]
Hi.

**CodeSandbox**
Just fill the two file inputs, could be the same file, doesn't matter..
[https://codesandbox.io/s/react-hook-form-controlled-input-bug-v0of1?file=/src/App.js](url)

**Problem**
When I submit the form, in the console the uncontrolled input give me the array of the files submitted, in the other hand, the controlled input gived me only a string with "C:/fakepath/filename", if I set the property multiple in the controlled input, I get only the path of one file.
Forgive my bad english

**Comment by bluebill1049** [comment, marked as answer]
for controlled input, you will need to manage your own state, it's probably getting e.target.value instead e.target.files

**Reply by gamino97** [reply]
That makes sense, thank you for answering.
